sentry-core/src/main/java/io/sentry/core/protocol/Device.java

```diff
@@ -12,10 +12,13 @@
   private String family;
   private String model;
   private String modelId;
-  private String architecture;
-  private Short batteryLevel;
-  private Boolean isCharging;
-  private Boolean isOnline;
+
+  @Deprecated private String architecture; // it was only arch, did we change it?
```

Comment:
It must serialize to `arch` yes.
The serialized names are documented here: https://docs.sentry.io/development/sdk-dev/event-payloads/